GitUp/GitUp.xcodeproj/project.pbxproj

```diff
@@ -335,6 +335,9 @@
 				DBBEE6BF256B0B1C00F96DAF /* PBXTargetDependency */,
 			);
 			name = Tool;
+			packageProductDependencies = (
+				0A4881DB26C7B98D00289CF9 /* Libgit2Origin */,
```

Comment:
Maybe this is a dumb question, but where does `Libgit2Origin` come from?

Reply:
Good question.
It comes from libgit2 package.
Oh, I have some misnaming convention here.


libgit2/ folder contains Package.swift. So, this folder can be added to GitUpKit.xcodeproj.
After adding this folder, Xcode allows to add targets from this libgit2/Package.swift.

Package.swift defines a set of targets. 
Package name is not used, because we are adding it locally by dragging it into xcodeproj.

```swift
let package = Package(
    name: "SwiftPackage", // Package may need a different name, but this name isn't used here, because we are adding it as local dependency.
    products: [
        .library(name: "Libgit2Origin", // This is a library itself. Let's name it later as libgit2?
                 targets: ["Libgit2Origin"]
        ),
        .library(name: "http-client",
                 targets: ["http-client"]
        ),
        .library(name: "ntlmclient",
                 targets: ["ntlmclient"]
        ),
    ])
```

Reply:
Xcode takes care of all these targets and prepares all artifacts in DerivedData.
These schemes can be found at Third-Party/libgit2/.swiftpm